src/vs/workbench/contrib/extensions/browser/extensionsWorkbenchService.ts

```diff
@@ -792,6 +802,32 @@ export class ExtensionsWorkbenchService extends Disposable implements IExtension
 		this.telemetryService.publicLog2<InstalledExtensionsEvent, ExtensionsLoadClassification>('installedExtensions', { extensionIds: extensionIds.join(';'), count: extensionIds.length });
 	}
 
+	private updateRunningExtensions(): void {
+		this.extensionService.getExtensions().then(async runningExtensions => {
+			const deltaExtensionsDescription: IExtensionDescription[] = [];
+			deltaExtensionsDescription.push(...runningExtensions.filter(e => !this.runningExtensions?.includes(e)));
```

Comment:
We shall use extension id to check for existence because we cannot rely that the extension service is returning the same object